### ISSUE TITLE
Classify Functionality for Multiple Segments

### ISSUE THREAD

**Issue by benneely** [duke-lungmap-team/ihc-image-analysis#50]
Currently, the `Classify` functionality allows users to segment multiple regions. However, when one presses the `Classify Region` button, only one result is presented. If we support multiple classify segments, need to change frontend. If we don't, need to implement some checking on the backend to ensure RESTful client adhere to same standard as our UI.

**Comment by whitews** [comment]
I also agree with this, however it becomes tricky to identify which polygon goes with which result in the modal, which is why I left it like that. Actually this becomes even more complicated as I think about it. There is no current identifier for a polygon, so once the points make the round trip back from the classify API we would need something like this to know which set of results belong to which set of points. drw-poly needs some sort of polygon ID implementation perhaps also with some labeling functionality.

**Comment by benneely** [comment]
Sounds like the easiest path forward (at this point) is to only support classification of one segment at a time? We should add a (UI) check before POSTing, since this is really a UI issue. We could still support multiple classifications at the RESTful API layer in the future if the need arises, but there is no demand at this point, so will keep that as is?

**Comment by whitews** [comment]
I'm wondering if there isn't a clever way to only allow a single region to be drawn...like if the user hits Enter to stop editing the first region, any subsequent click would be ignored until the first region is deleted?
